<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-SCTI6QKF" ignored="0" name="Test" tests="4" failures="2" timestamp="2022-08-05T15:12:47 IST" time="19.968" errors="0">
  <testcase name="InvalidLogin3" time="0.0" classname="com.obsqura.scripts.TestClass">
    <failure type="org.testng.TestNGException" message="
Method public void com.obsqura.scripts.TestClass.InvalidLogin3() throws java.io.IOException requires a @DataProvider named : pass">
      <![CDATA[org.testng.TestNGException: 
Method public void com.obsqura.scripts.TestClass.InvalidLogin3() throws java.io.IOException requires a @DataProvider named : pass
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- InvalidLogin3 -->
  <testcase name="InvalidLogin2" time="3.755" classname="com.obsqura.scripts.TestClass">
    <failure type="java.lang.AssertionError" message="expected [Epic sadface: Password is required] but found [Epic sadface: Password is required1]">
      <![CDATA[java.lang.AssertionError: expected [Epic sadface: Password is required] but found [Epic sadface: Password is required1]
at com.obsqura.scripts.TestClass.InvalidLogin2(TestClass.java:74)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- InvalidLogin2 -->
  <testcase name="InvalidLogin1" time="0.862" classname="com.obsqura.scripts.TestClass"/>
  <testcase name="verifyLogin" time="0.704" classname="com.obsqura.scripts.TestClass"/>
</testsuite> <!-- Test -->
